Webitools 112 creating application if not exist should provide possibility to select a css server lfo (#102)
CAST-Extend/com.castsoftware.aip.console.tools: aip-console-tools-cli/README.md

- `-n` or `--name` (**required**) : specify the name of the application to create

* `--node-name` (optional) : specify the name of an AIP Node on which the application will be create. *default* :
  Automatically selected by AIP Console
* `--domain-name` (optional) : specify a domain name to associate with the application to create. *default*: No domain
  is set for the application.
* `--inplace-mode` (optional):  If true then no history will be kept for delivered sources.
* `--verbose` (optional): Whether the command log should be output to the console or not, defaulted to true
* `--server-url` or `-s` : Specify the URL to your AIP Console server. *default* : localhost:8081
* `--apikey` or `--apikey:env` (either is required) : the API Key to log in to AIP Console **OR** the environment
  variable containing the key
* `--timeout` (optional) : Time in seconds before calls to AIP Console time out. *default* : 90
* `--user` (optional) (legacy) : Specify a username to log in. Requires passing the user's password in the `--apikey`
  parameter. *default* : none

> * `--css-server` or `-css` (optional): CSS server you which to host the created application' data.

Reviewer: ```suggestion
* `--css-server` or `-css` (optional): CSS Server name where the application data will be stored.
```
